src/Types.hs

```diff
@@ -354,6 +354,11 @@ fbSearchPathKeybindings = lens _fbSearchPathKeybindings (\cv x -> cv { _fbSearch
 fbHomePath :: Lens (FileBrowserSettings a) (FileBrowserSettings a') a a'
 fbHomePath = lens _fbHomePath (\s a -> s { _fbHomePath = a })
 
+data Delay
```

Comment:
This looks like a stray addition from https://github.com/purebred-mua/purebred/commit/44946bfe62eb8288fd2cf45ef43200de59ca0a81 ?

Reply:
nah I just moved it.  it was in the way :)  see removed lines below.

Reply:
Ah dang!